The SLA Credit Process for CenturyLink Cloud Customers

```diff
@@ -0,0 +1,23 @@
+Description (goal/purpose)
```

Comment:
@dcline18 You are missing the `front-matter` at the top of the file.  Please see any other KB article for the required formatting.

Also, you should probably add headers `### How do I Blah?` for your questions.